ueransim/gnb/gnb-configmap.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: gnb-configmap
+  labels:
+    app.kubernetes.io/version: "v3.2.6"
+    app: ueransim
+    component: gnb
+data:
+  gnb-config.yaml: |
+    mcc: '208'          # Mobile Country Code value
+    mnc: '93'           # Mobile Network Code value (2 or 3 digits)
+    nci: '0x000000010'  # NR Cell Identity (36-bit)
+    idLength: 32        # NR gNB ID length in bits [22...32]
+    tac: 1              # Tracking Area Code
+    # List of supported S-NSSAIs by this gNB
+    slices:
+      - sst: 0x1
+        sd: 0x010203
+    # Indicates whether or not SCTP stream number errors should be ignored.
+    ignoreStreamIds: true
+    
+    linkIp: 0.0.0.0   # gNB's local IP address for Radio Link Simulation (Usually same with local IP)
+    # gNB's local IP address for N2 Interface (Usually same with local IP)
+    ngapIp: 10.100.50.250
+    gtpIp: 10.0.0.20    # gNB's local IP address for N3 Interface (Usually same with local IP)
```

Comment:
How do we handle these IPs in different environments?

Reply:
Let's merge as is - the README tells how to set these. We can update it later. We have a few options:

0) Documentation (the existing way in the README).
1) If we can automatically figure it out, or maybe point it at a topology resource and figure it out from there, that would be ideal. We would use the Nephio primitives (IPAM/NAD funcs, for example).
2) We can create an in-package CR (ie, a version package-specific API) and a function to populate these values in the right places. This could, for example, provide a network name rather than a specific IP, and combine with 1) to configure the package more easily.
3) We can use kpt setters.
4) We can use values in the package-context.yaml ConfigMap and a function to populate these values in the right places.

So, for the initial commit I think this is good, we will evolve from there.

/approve

Reply:
would we not specialise these IP(s) through the pipeline?

Reply:
Yes, most likely.

Reply:
(that's what I meant by 1 above)